app/javascript/components/common/Accordion.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+
+export type AccordionContext = {
+  index: string
```

Comment:
I was slightly confused about `index` being of type `string`. With `index` I immediately assumed it was a number. Maybe a different name can be used, like `id` or `identifier` or something like that?

Reply:
Cool. I've used `id` for this.